pool/connection_pool_test.go

```diff
@@ -1115,15 +1115,15 @@ func TestConnectionHandlerOpenUpdateClose(t *testing.T) {
 	poolInstances := makeInstances(poolServers, connOpts)
 	roles := []bool{false, true}
 
-	ctx, cancel := test_helpers.GetPoolConnectContext()
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
 	defer cancel()
 
 	err := test_helpers.SetClusterRO(ctx, makeDialers(poolServers), connOpts, roles)
-	require.Nilf(t, err, "fail to set roles for cluster")
+	require.NoError(t, err, "fail to set roles for cluster")
 
 	h := &testHandler{}
 	poolOpts := pool.Opts{
-		CheckTimeout:      100 * time.Microsecond,
+		CheckTimeout:      100 * time.Millisecond,
```

Comment:
Does the change required? Or is it enough to just increase the timeout for context?